.github/workflows/on-workflow-call-www-deployment.yml

```
@@ -16,30 +20,29 @@ permissions:
 jobs:
   deploy:
-    environment: preview
     runs-on: ubuntu-latest
+    environment: ${{ inputs.deployment_environment }}
+
     steps:
       - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-env
+        with:
+          ENV_SERVICE_TOKEN: ${{ secrets.ENV_SERVICE_TOKEN }}
       - uses: ./.github/actions/vercel-build
```

actionlint (1.7.7)

missing input "ENV_SERVICE_TOKEN" which is required by action "Vercel Build" defined at "./.github/actions/vercel-build". all required inputs are "ENV_SERVICE_TOKEN", "VERCEL_ORG_ID", "VERCEL_PROJECT_ID", "VERCEL_TOKEN", "project_name"

(action)